charts/operator-wandb/charts/weave/templates/deployment.yaml

```
@@ -55,6 +55,8 @@ spec:
               value: weave
             - name: WANDB_BASE_URL
               value: {{ .Values.global.host }}
+            - name: WANDB_PRIVATE_BASE_URL
+              value: http://{{ include "weave.appFullname" . }}.{{ $.Release.Namespace }}.svc.{{ .Values.app.clusterDomain }}:8080
```

```suggestion
              value: http://{{ include "weave.appFullname" . }}:8080
```

Why cannot we keep this simple?